### ISSUE TITLE
TypeError: Cannot read property 'apply' of undefined

### ISSUE THREAD

**Issue by moshest** [lifaon74/url-polyfill#46]
Got the following error:

```
TypeError: Cannot read property 'apply' of undefined
  apply (webpack:///node_modules/url-polyfill/url-polyfill.js:305:0),
```

Raised when calling `url.searchParams.set(key, value);`

User agents:
```
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36
```

**Comment by lifaon74** [comment]
Could you please debug and do a pull request as I can't reproduce your problem ?

Check if *global.URLSearchParams* exists (should), and that it has 'append', 'delete' and 'set' properties. If not you may have an imcomplete polyfill of `URLSearchParams` somewhere.

**Comment by moshest** [comment]
I try to debug it but I can't reproduce it as well. I got those errors from clients.

No incomplete polyfill of `URLSearchParams` found, and as far as I know, Chrome/73+ should have native implementation of URL and URLSearchParams so the polyfill should never run on those machines.

**Comment by lifaon74** [comment]
I close the issue. If you find any solution, don't hesitate to re-open.

**Comment by moshest** [comment]
Maybe relate: https://github.com/jerrybendy/url-search-params-polyfill/issues/34

It's possible to add test for `global.URLSearchParams.prototype.set`?

Also, from looking at the code, I would put the check for `URLSearchParams` in `try..catch` block.

**Comment by moshest** [comment]
See: https://github.com/lifaon74/url-polyfill/pull/48

**Comment by lifaon74** [comment]
released as 1.1.7
